lib/push.js

```diff
@@ -83,79 +117,161 @@ module.exports = function (log, db) {
     },
 
     /**
+     * Notifies all devices (except the one who joined) that a new device joined the account
+     *
+     * @param uid
+     * @param deviceName
+     * @param currentDeviceId
+     * @promise
+     */
+    notifyDeviceConnected: function notifyDeviceConnected(uid, deviceName, currentDeviceId) {
+      var data = new Buffer(JSON.stringify({
+        version: PUSH_PAYLOAD_SCHEMA_VERSION,
+        command: pushCommands.deviceConnected,
+        data: {
+          deviceName: deviceName
+        }
+      }))
+      var options = { data: data, excludedDeviceIds: [currentDeviceId] }
+      return this.pushToDevices(uid, 'deviceConnected', options)
+    },
+
+    /**
+     * Notifies a device that it is now disconnected from the account
+     *
+     * @param uid
+     * @param idToDisconnect
+     * @promise
+     */
+    notifyDeviceDisconnected: function notifyDeviceDisconnected(uid, idToDisconnect) {
+      var data = new Buffer(JSON.stringify({
+        version: PUSH_PAYLOAD_SCHEMA_VERSION,
+        command: pushCommands.deviceDisconnected,
+        data: {
+          id: idToDisconnect
+        }
+      }))
+      var options = { data: data, TTL: TTL_DEVICE_DISCONNECTED }
+      return this.pushToDevice(uid, idToDisconnect, 'deviceDisconnected', options)
+    },
+
+    /**
      * Send a push notification with or without data to all the devices in the account (except the ones in the excludedDeviceIds)
      *
      * @param uid
      * @param reason
      * @param data
-     * @param excludedDeviceIds
+     * @param options {excludedDeviceIds, data, TTL}
```

Comment:
where is this options format from?

I've seen the expanded one from the docs, e.g:
```
 * @param {Object} options
 * @param {String} options.name    A person's name
 * @param {String} options.feature A person's property
```